mindjunkies/dashboard/views.py

```
@@ -105,8 +163,35 @@ def form_invalid(self, form):
+class ArchiveView(LoginRequiredMixin, View):
+    permission_required = VIEW_COURSE_PERMISSION
+
+    def get(self, request: HttpRequest) -> HttpResponse:
+        if not request.user.is_teacher:
+            return redirect("teacher_verification_form")
+
+        courses = Course.objects.filter(teacher=request.user, status="draft")
+        context = {
+            "courses": courses,
+        }
+        return render(request, "components/archive.html", context)    
```

Ruff (0.8.2)

Trailing whitespace

Remove trailing whitespace

(W291)